app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughActivityPresenter.kt

```diff
@@ -138,4 +138,12 @@ class WalkthroughActivityPresenter @Inject constructor(
   fun setTopicId(topicId: String) {
     this.topicId = topicId
   }
+
+  fun hideProgressBarAndShowHeader() {
+    getWalkthroughViewModel().hideProgressBarAndShowHeader()
```

Comment:
As per my earlier thought, creating a global ViewModel var we can use it everything rather then calling getViewModel.

Reply:
@anandwana001 Should I update it everywhere in the file?

Reply:
yes, I guess. 
1. create a variable for this viewmodel. 
2. use that variable at all required places.

Reply:
@anandwana001 var name ```walkthroghViewModel``` is giving same JVM signature error and name ```viewModel``` can't be used, see [this]() comment. So I have used ```mViewModel``` for now, we can discuss on a better name and fix it.